packages/core/src/components/forms/inputGroup.tsx

```diff
@@ -91,6 +91,15 @@ export class InputGroup extends React.Component<HTMLInputProps & IInputGroupProp
         this.updateInputWidth();
     }
 
+    private maybeRenderLeftIcon() {
+        const { leftIconName } = this.props;
+        if (leftIconName == null) {
+            return null;
```

Comment:
Noticed that but didn't want to both refactor and change behavior. Just committed that fix though.